src/lib/core/WeaveConfig.h

```diff
@@ -1904,6 +1904,23 @@
 #define WEAVE_CONFIG_ENABLE_SERVICE_DIRECTORY               1
 #endif // WEAVE_CONFIG_ENABLE_SERVICE_DIRECTORY
 
+/**
+ *  @def WEAVE_CONFIG_PERSIST_SERVICE_DIRECTORY
+ *
+ *  @brief
+ *    If set to (1), persistent service directory is enabled.
+ *    Default value is (0) or disabled.
+ *
+ *  @note
+ *    Enabling this profile persists service directory after
```

Comment:
Recommend rephrasing as "Enabling this feature flag persists the service directory cache retrieved from the directory server and use it to populate the in-memory cache at initialization time.